### PR TITLE
Some fixes for the release process and 0.6.0

### PR DESCRIPTION
**Pull request by krnowak**
I did some gitgrepping for old versions and found some cases not covered by `make update-version` so I just added them. In the long run we will want to have something better than this.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Habitat operator CHANGELOG
 
+## [v0.6.0](https://github.com/kinvolk/habitat-operator/tree/v0.6.0) (20-4-2018)
+[Full changelog](https://github.com/kinvolk/habitat-operator/compare/v0.5.1...v0.6.0)
+
+### Features & Enhancements
+
+- New "version" of the habitat CRD that uses StatefulSets, has persistence functionality. Previous version that uses Deployments is still supported but is frozen [#201](https://github.com/habitat-sh/habitat-operator/pull/201) [#240](https://github.com/habitat-sh/habitat-operator/pull/240)
```
Comment (iaguis) [inline CHANGELOG.md]: Previous -> "The previous"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+- Oldest supported Habitat Supervisor is 0.52 [#190](https://github.com/habitat-sh/habitat-operator/pull/190)
+
+Please compare `v1beta1` and `v1beta2` manifest of the standalone example in examples/v1beta1/habitat.yml and examples/standalone/habitat.yml, respectively, to compare the immediate differences between them. Please refer to examples/persisted/habitat.yml for an example of the persistence functionality.
```
Comment (iaguis) [inline CHANGELOG.md]: s/Please compare `v1beta1` and `v1beta2` manifest/Please compare the `v1beta1` and `v1beta2` manifests/

**Comment by krnowak** [comment]
Updated.

File: helm/habitat-operator/README.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ The following table lists the configurable parameters of the habitat-operator ch
 Parameter | Description | Default
 --- | --- | ---
 `image.repository` | Image | `docker.io/habitat/habitat-operator`
-`image.tag` | Image tag | The latest release tag (e.g `v0.4.0`)
+`image.tag` | Image tag | The latest release tag (e.g `v0.6.0`)
```
[the remark this answers is not in the document]
Comment (asymmetric) [reply helm/habitat-operator/README.md]: Ah I see, they were automatically changed by the updated make target.

**Pull request merged**